package/src/mock-builders/mock.js

```diff
@@ -17,23 +17,36 @@ const setUser = (client, user) =>
     resolve();
   });
 
-function mockClient(client) {
+function mockClient(client, options = {}) {
+  const { disableAppSettings = true } = options;
+
   jest.spyOn(client, '_setToken').mockImplementation();
   jest.spyOn(client, '_setupConnection').mockImplementation();
   client.tokenManager = {
     getToken: jest.fn(() => token),
     tokenReady: jest.fn(() => true),
   };
   client.setUser = setUser.bind(null, client);
+
+  if (disableAppSettings) {
+    client.getAppSettings = jest.fn(() => ({}));
+  }
```

Comment:
Disabling this `getAppSettings` call since it gets executed in every test where we render `Chat` component. And it will result in error since there is no real api key and stuff. Solution would be to mock this api call, but I think instead of patching it for every test I think it simply makes sense to supress this api call, unless we are actually trying to test `getAppSettings` implementation. One can still enable this api call by setting `disableAppSettings: false`

Reply:
@Enigma-I-am CC